frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -101,83 +80,129 @@ export function ProjectList() {
   const [currentPage, setCurrentPage] = React.useState(1);
   const [perPage, setPerPage] = React.useState(10);
 
+  const query = useProjectList();
+
+  if (query.isLoading) {
+    return <Loader />;
+  }
+  const projectList = query.data?.data?.results;
```

Comment:
```suggestion
  const projectList = query.data?.data?.results ?? [];
```

You will have a TS error otherwise

Reply:
You also need to run this in the frontend to update the api sdk file (`src/api/v1.ts`): 

```shell
pnpm generate:api 
```